corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -103,12 +105,20 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None):
             if getattr(f, 'schedule', False) and f.schedule.enabled
         ])
 
-
+    vellum_base = 'corehq/apps/app_manager/static/app_manager/js/'
+    vellum_dir = 'vellum'
+    if toggles.VELLUM_BETA.enabled(domain) and isdir(join(vellum_base, 'vellum_beta')):
+        vellum_dir = 'vellum_beta'
     context = get_apps_base_context(request, domain, app)
     context.update(locals())
     context.update({
-        'vellum_debug': settings.VELLUM_DEBUG,
+        'vellum_debug': settings.VELLUM_DEBUG and not toggles.VELLUM_BETA,
```

Comment:
Do I understand correctly that `VELLUM_BETA` overrides `VELLUM_DEBUG`? That seems a little odd. I'd think we'd want the opposite, although maybe it doesn't really matter.